runtime-modules/content/src/lib.rs

```diff
@@ -2921,7 +2945,15 @@ decl_module! {
         }
 
         /// Channel collaborator remark
-        #[weight = 10_000_000] // TODO: adjust weight
+        /// <weight>
+        ///
+        /// ## Weight
+        /// `O (1)`
+        /// - DB:
+        ///    - O(B) where:
+        ///        - B is the byte lenght of `msg`
```

Comment:
Weight should be `O(B)`, but DB should be `O(1)` (no db reads/writes are dependent on the weight function parameters)